txpool/txpool_test.go

```diff
@@ -3627,6 +3730,86 @@ func TestGetTxs(t *testing.T) {
 	}
 }
 
+func TestFreeSlots(t *testing.T) {
+	pool, err := newTestPool()
+	require.NoError(t, err)
+
+	freeslot := pool.gauge.freeSlots()
+	require.NotEqualValues(t, uint64(0), freeslot)
```

Comment:
```suggestion
	require.Greater(t, freeslot, uint64(0))
```
Maybe to use Greater instead of NotEqualValue. It is more appropriate for this test since we are expecting some value greater than 0.